OpenSim/Common/DelimFileAdapter.h

```diff
@@ -400,41 +403,51 @@ DelimFileAdapter<T>::extendRead(const std::string& fileName) const {
                      _timeColumnLabel,
                      column_labels[0]);
     column_labels.erase(column_labels.begin());
-    // Set the column labels as metadata.
-    ValueArray<std::string> value_array{};
-    for(const auto& cl : column_labels)
-        value_array.upd().push_back(SimTK::Value<std::string>{cl});
-    typename TimeSeriesTable_<T>::DependentsMetaData dep_metadata{};
-    dep_metadata.setValueArrayForKey("labels", value_array);
-    table->setDependentsMetaData(dep_metadata);
 
     // Read the rows one at a time and fill up the time column container and
     // the data container.
+    std::vector<double> timeVec;
+    std::vector<SimTK::RowVector_<T>> depDataVec;
+    int initCapacity = 2500;
```

Comment:
Could you add a comment for how you chose this number? I would lean towards not reserving memory initially, especially if it doesn't have a big impact on the performance test you ran.